sunpy/net/dataretriever/sources/tests/test_noaa.py

```diff
@@ -227,6 +222,13 @@ def test_srs_start_or_end_out_of_range(srs_client):
     assert len(res) > 0
 
 
+@pytest.mark.remote_data
+def test_tar_file_broken():
+    # 2010 extracts out to 2010_SRS while other years do SRS only.
+    results = Fido.search(a.Time("2010/5/1", "2010/10/28"), a.Instrument.soon)
+    results = Fido.fetch(results)
+
```

Comment:
assert that results actually contains something?

Reply:
Fair.

Reply:
Fixed.